jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/Main.java

```diff
@@ -64,13 +64,13 @@ public static Map<String, Object> shipData(Map<String, String> params) {
 		}
         String ds = params.get("datasource");
         ESManager.configureIndexAndTypes(ds,errorList);
-        errorList.addAll(new EntityManager().uploadEntityData(ds));
-        ExternalPolicies.getInstance().uploadPolicyDefinition(ds);
-        if("aws".equals(ds)) {
-        	errorList.addAll(new AssetGroupStatsCollector().collectAssetGroupStats());
-        	errorList.addAll(new IssueCountManager().populateViolationsCount());
-        	errorList.addAll(new AssetsCountManager().populateAssetCount());
-        }
+//        errorList.addAll(new EntityManager().uploadEntityData(ds));
```

Comment:
@raviteja-mandala please remove commented code. Its bad practice. We should always be able to lookup previous implementation from history. If this code is going to production, then please remove any such comments.

Reply:
okay @plyubich .